etna/transforms/math/scalers.py

```diff
@@ -145,6 +171,27 @@ def __init__(
             mode=mode,
         )
 
+    def params_to_tune(self) -> Dict[str, "BaseDistribution"]:
+        """Get default grid for tuning hyperparameters.
+
+        This grid tunes parameters: ``mode``, ``with_centering``, ``with_scaling``, ``unit_variance``.
+        Other parameters are expected to be set by the user.
+
+        Returns
+        -------
+        :
+            Grid to tune.
+        """
+        grid = super().params_to_tune()
+        grid.update(
+            {
+                "with_centering": CategoricalDistribution([False, True]),
+                "with_scaling": CategoricalDistribution([False, True]),
+                "unit_variance": CategoricalDistribution([False, True]),
```

Comment:
I tried to add `quantile_range`, it worked, but I don't really sure should we add it here because of the warning.